### PR TITLE
Add guide for Pulumi CDK

### PR DESCRIPTION
**Pull request by corymhall**
This PR adds a guide to using Pulumi CDK. This seemed like the best place to put this content, but if there is a better place let me know!

I plan on creating a separate migration guide which I will then update this to link to.

re #250

### CODE REVIEW

**Comment by corymhall** [comment]
Link is here http://www-testing-pulumi-docs-origin-pr-13590-067915ff.s3-website.us-west-2.amazonaws.com/docs/iac/clouds/aws/guides/cdk/

**Comment by t0yv0** [comment]
I think this could use a link to construct hub and some top constructs close to the top of the article! Otherwise LGTM.

Needs @pulumi/docs to pipe in where this should go though.

File: content/docs/iac/clouds/aws/guides/cdk.md [hunk 1]
```diff
@@ -0,0 +1,834 @@
...
+```console
+
+cdk:construct:StagingStack (staging-stack):
+    error: Duplicate resource URN 'urn:pulumi:project::pulumi-lookups-enabled::cdk:index:App$cdk:construct:StagingStack::staging-stack-'; try giving it a unique name
+```
+
+At this point the lookups have been performed and you should be able to run
+Pulumi commands without errors.
+
+## Using Pulumi Policy Packs
```
Comment (flostadler) [inline content/docs/iac/clouds/aws/guides/cdk.md:461]: Nice!

**Review by flostadler — COMMENTED** [review]
Looks great! Left some minor comments

**Review by thoward — APPROVED** [review]
Looks good. Thanks @corymhall !

**Pull request merged**